kolibri/core/assets/src/views/AppError/index.vue

```diff
@@ -51,9 +49,30 @@
 
 <script>
 
+  import get from 'lodash/get';
+  import urls from 'kolibri.urls';
   import { mapActions } from 'vuex';
   import ReportErrorModal from './ReportErrorModal';
 
+  // Inspects URL and returns an enum of the plugin (enum values are "learn", "coach", etc.)
+  function getCurrentKolibriPlugin() {
+    const url = window.location.href;
+
+    if (url.includes(urls['kolibri:learn:learn']())) {
+      return 'LEARN';
+    } else if (url.includes(urls['kolibri:coach:coach']())) {
```

Comment:
@rtibbles These URL names are probably wrong now. Can you share off the top of your head what they should be now?